CMake 3.12 errors on FOLLOW_SYMLINKS (which was never supported for
file(GLOB ...) anyway, but was ignored).

See also https://springrts.com/mantis/view.php?id=6005

--- ./rts/build/cmake/UtilJava.cmake.bak	2014-10-07 22:09:51 UTC
+++ ./rts/build/cmake/UtilJava.cmake
@@ -45,7 +45,7 @@
 # Returns the name of the first sub-dir (in alphabetical descending order)
 # under dir.
 macro    (GetFirstSubDirName name_var dir)
-	file(GLOB dirContent RELATIVE "${dir}" FOLLOW_SYMLINKS "${dir}/*")
+	file(GLOB dirContent RELATIVE "${dir}" "${dir}/*")
 	foreach    (dirPart ${dirContent})
 		if    (IS_DIRECTORY "${dir}/${dirPart}")
 			set(${name_var} ${dirPart})
